src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -659,6 +659,48 @@ void CheckSoftminDerivativeCorrect(const arma::colvec input,
   }
 }
 
+/**
+ * Implementation of the Flatten T Swish activation function test. The function is
+ * implemented as Flatten T Swish layer in the file flatten_t_swish.hpp.
+ *
+ * @param input Input data used for evaluating the Flatten T Swish activation function.
+ * @param target Target data used to evaluate the Flatten T Swish activation.
+ */
+void checkFlattenTSwishActivationCorrect(const arma::colvec input, const arma::colvec target)
```

Comment:
We use upper camel casing, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions for details.